lms/djangoapps/commerce/urls.py

```diff
@@ -4,10 +4,10 @@
 
 from django.conf.urls import patterns, url
 
-from .views import OrdersView, checkout_cancel
+from .views import BasketsView, checkout_cancel
 
 urlpatterns = patterns(
     '',
-    url(r'^orders/$', OrdersView.as_view(), name="orders"),
+    url(r'^baskets/$', BasketsView.as_view(), name="baskets"),
```

Comment:
@wedaly thanks, both good calls.
I will look at making an adapter to keep orders/ working with the v2 api.
The bok choy failures actually traced back to that same line in enrollment.js, I've fixed that.